src/Microsoft.AspNet.Mvc.Razor.Host/Directives/ChunkInheritanceUtility.cs

```diff
@@ -40,17 +40,18 @@ public class ChunkInheritanceUtility
         }
 
         /// <summary>
-        /// Gets an ordered <see cref="IReadOnlyList{T}"/> of parsed <see cref="ChunkTree"/> for each
-        /// <c>_ViewImports</c> that is applicable to the page located at <paramref name="pagePath"/>. The list is
-        /// ordered so that the <see cref="ChunkTree"/> for the <c>_ViewImports</c> closest to the
+        /// Gets an ordered <see cref="IReadOnlyList{ChunkTreeResult}"/> of parsed <see cref="ChunkTree"/>s and
```

Comment:
FYI `<see cref="IReadOnlyList{T}"/>` and `<see cref="IReadOnlyList{ChunkTreeResult}"/>` are displayed the same. That's why these comments include bits like `of parsed <see cref="ChunkTree"/>`.